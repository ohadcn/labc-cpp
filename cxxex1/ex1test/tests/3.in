100,100
200,200
5,7
