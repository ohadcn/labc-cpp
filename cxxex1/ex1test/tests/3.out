result
5,7
100,100
200,200
